src/components/ModalPortal.js

```diff
@@ -119,9 +119,8 @@ export default class ModalPortal extends Component {
 
   beforeOpen() {
     const { appElement, ariaHideApp, bodyOpenClassName } = this.props;
-    refCount.add(bodyOpenClassName);
     // Add body class
-    elementClass(document.body).add(bodyOpenClassName);
+    bodyClassList.add(bodyOpenClassName);
```

Comment:
@diasbruno, which implementation would you prefer? Does the thumbs up mean you think I should implement the code outlined in the commit I linked to? I think the code in the external commit is a more robust solution than what currently exists in this PR, but it's up to you :)